### PR TITLE
EOS-18521:Bazel enhancements and kafka script modification

### PR DESCRIPTION
**Pull request by sachitanands**
1) Enable option to specify ram and cpu usage for bazel during jenkins-build.sh script
2) Fix issue with install-kafka script

Signed-off-by: Sachitanand Shelake <sachitanand.shelake@seagate.com>

### CODE REVIEW

File: rebuildall.sh [hunk 1]
```diff
@@ -195,12 +199,16 @@ while true; do
     --no-java-tests) no_java_tests=1; shift ;;
     --just-gen-build-file) just_gen_build_file=1; shift ;;
     --valgrind_memcheck) valgrind_memcheck=1; shift ;;
+    --bazel_cpu_usage_limit) bazel_cpu_usage_limit=$2; shift ;;
+    --bazel_ram_usage_limit) bazel-rpm-usage-limit=$2; shift ;;
     -h|--help) usage; exit 0;;
     --) shift; break ;;
     *) echo "Internal error!" ; exit 1 ;;
   esac
+  echo "iterating logic"
```
Comment (nileshgovande) [inline rebuildall.sh:208]: Remove

File: rebuildall.sh [hunk 2]
```diff
@@ -195,12 +199,16 @@ while true; do
...
 done
 
+exit 1
```
Comment (nileshgovande) [inline rebuildall.sh:211]: Remove this

**Review by nileshgovande — APPROVED** [review]

**Comment by nileshgovande** [comment]
Additional jobs 
http://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/centos-7.8/job/cortx-custom-ci/2214/ 
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Component-Test/job/S3Server-PreMerge-Test/1878/

**Review by vimalraghwani20 — APPROVED** [review]

**Comment by nileshgovande** [comment]
All jobs passed

**Review by sachitanands — COMMENTED** [review]
looks good

File: rebuildall.sh (hunk 2 above)
Comment (sachitanands) [reply rebuildall.sh]: done

File: rebuildall.sh (hunk 1 above)
Comment (sachitanands) [reply rebuildall.sh]: done

**Pull request merged**